cmd/simulacra/simulacra.go

```diff
@@ -83,11 +83,20 @@ type Config struct {
 	ImageProject string
 }
 
-func distroFolder(platform string) (string, error) {
-	if gce.IsWindows(platform) {
+func distroFolder(imageSpec string) (string, error) {
```

Comment:
I don't have a good place to leave this comment, but please remember to pin google3 tests before merging this PR.

Reply:
Done